### PR TITLE
Recipe #0230—Chronology with navDate

### PR DESCRIPTION
**Pull request by triplingual**
This PR addresses issue #230, introducing the way to communicate date information to a consuming client. This property and its value are not intended to be displayed to end viewers.

### CODE REVIEW

**Comment by glenrobson** [comment]
Hi Trip,

I'm afraid the cookbook editors had difficulty with this one. Nothing to do with the way the recipe was written, as all agreed that the writing looked good. The difficulty was in the example and the fact it doesn't work in viewers currently and where the navDate is currently located in the canvas. We could think of a few use cases where navDate could be useful in a canvas but so far nobody has pushed for this to be implemented in any of the viewers and so there was discussion on whether this recipe would be useful to drive implementation or if it would be a use case which is spec complaint but less likely to be one people would use. There was also discussion on whether the canvas navDate use cases would be better served by a navDate in a range.

To avoid this issue of canvas navDate would you be OK to move to a collection with 2 manifests all with navDates (at manifest level in the manifests and in the manifest object in the collection)? I can upload another LOC image to fixtures. This will hopefully work with the UV and it will be a useful recipe that focuses on just the navDate property rather than the multiple newspaper related features in the Newspaper recipe. Does this sound OK?

A couple of other notes:
 * Chronology with navDate - could the title be changed to something like "Navigating in chronological order" so as not to have the property name in the title. 
 * Could you cut down info.json of the image to just the `id`, `type` and `profile`. They felt that otherwise it distracted from the recipe focus (I did pass on that the full info.json is better for efficiency reasons) 
 
If your OK to do the changes I think we can still get this recipe to the TRC in two weeks. Once you've done the changes Ill ask the editors to comment on it through slack rather than a meeting.

**Comment by triplingual** [comment]
Can do, @glenrobson

**Comment by triplingual** [comment]
✅ "could the title be changed to something like "Navigating in chronological order" so as not to have the property name in the title"
✅ "Could you cut down info.json of the image to just the id, type and profile."

File: _includes/links.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
 [0068]: {{site.baseurl}}/recipe/0068-newspaper/ "A newspaper"
 [0117]: {{site.baseurl}}/recipe/0117-add-image-thumbnail/ "Manifest Thumbnail"
+[0230]: {{site.baseurl}}/recipe/0230-navdate/ "Chronology with navDate"
```
Comment (glenrobson) [inline _includes/links.md:27]: Could you update this with the new recipe title?

Comment (triplingual) [reply _includes/links.md:27]: Yes indeed. While I'm at it, is that a good section for this recipe? My apologies for not raising it at the last authors' call.

**Comment by glenrobson** [comment]
There looks to be some issues with the view of the json in the preview: https://preview.iiif.io/cookbook/0230-navdate/recipe/0230-navdate/

Do you know whats up? I'm breaking for dinner now but if you want me to take a look let me know. Do you need me to upload another image to fixtures?

**Comment by triplingual** [comment]
And yes, @glenrobson, if you would add [https://www.loc.gov/resource/g3842c.ct008614/](https://www.loc.gov/resource/g3842c.ct008614/) to Fixtures

**Comment by glenrobson** [comment]
Here is the fixture: https://fixtures.iiif.io/info.html?file=/images/loc/chesapeake_map/87691274-1986.jpg

File: recipe/0230-navdate/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Restrictions
+
+None known.
+
+## Example
+
+This recipe presents an imaginary Collection containing 2 instances from the run of a particular national park map held at the United States' Library of Congress. If you want to expand this example locally, you may find other maps in this series, at the [Library of Congress website](https://www.loc.gov/maps/?q=Chesapeake+and+Ohio+Canal&fa=contributor%3Aunited+states.+national+park+service&st=list&c=100).
+
+{% include manifest_links.html viewers="UV, Mirador" manifest="navdate-collection.json" %}
```
Comment (glenrobson) [inline recipe/0230-navdate/index.md:31]: Could you put the links below the collection header?

File: recipe/0230-navdate/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Collection
+
+{% include jsonviewer.html src="navdate-collection.json" config='data-line="19"' %}
+
+### 1986 Map
+
```
Comment (glenrobson) [inline recipe/0230-navdate/index.md:38]: Could you include the manifest_links.html include here?

File: recipe/0230-navdate/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+{% include jsonviewer.html src="navdate_map_2-manifest.json" config='data-line="10"' %}
+
+### 1987 Map
+
```
Comment (glenrobson) [inline recipe/0230-navdate/index.md:42]: and here if thats OK.

File: _includes/links.md (hunk 1 above)
Comment (glenrobson) [reply _includes/links.md]: Maybe leave it where it is for now but I think we need to look at the headings now we have so many done in the first section.

File: recipe/0230-navdate/index.md (hunk 2 above)
Comment (triplingual) [reply recipe/0230-navdate/index.md:31]: I've placed a `manifest_links.html` include under each header, with the `manifest` attribute value of the adjacent Manifest, if that's how you meant this.

File: recipe/0230-navdate/index.md (hunk 3 above)
Comment (triplingual) [reply recipe/0230-navdate/index.md:38]: Added

File: recipe/0230-navdate/index.md (hunk 4 above)
Comment (triplingual) [reply recipe/0230-navdate/index.md:42]: Added

**Comment by glenrobson** [comment]
Perfect thanks Trip!

**Comment by glenrobson** [comment]
Approved with no changes requested by TRC.

**Pull request merged**